src/javascript/binary/common_functions/form_validation.js

```diff
@@ -9,6 +9,7 @@ const compareBigUnsignedInt = require('../common_functions/string_util').compare
 const Validation = (() => {
     const forms        = {};
     const error_class  = 'error-msg';
+    const error_address_class = 'error-address-msg';
```

Comment:
can you align line 10 to 13 by the `=` sign?

Reply:
Will do.